src/coreComponents/constitutive/relativePermeability/BrooksCoreyBakerRelativePermeability.cpp

```diff
@@ -168,7 +168,8 @@ BrooksCoreyBakerRelativePermeability::createKernelWrapper()
                         m_phaseTypes,
                         m_phaseOrder,
                         m_phaseRelPerm,
-                        m_dPhaseRelPerm_dPhaseVolFrac );
+                        m_dPhaseRelPerm_dPhaseVolFrac,
+                        m_phaseTrapped );
```

Comment:
```suggestion
                        m_phaseTrappedVolFrac );
```
for consistency with the variable name just above.

Maybe for all the relative permeability classes, you can use `phaseTrappedVolFrac` instead of just `phaseTrapped` if that is ok with you.